src/parser/csql_grammar.y

```diff
@@ -3757,21 +3757,51 @@ alter_stmt
 			    node->info.serial.cached_num_val = cached_num_val;
 			    node->info.serial.no_cache = no_cache;
 			    node->info.serial.comment = comment;
+			    node->info.serial.code = PT_SERIAL_OPTION;
 			  }
 
 			$$ = node;
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
 			if (!start_val && !increment_val && !max_val && !min_val
 			    && cyclic == 0 && no_max == 0 && no_min == 0
-			    && no_cyclic == 0 && !cached_num_val && no_cache == 0
-			    && comment == NULL)
+			    && no_cyclic == 0 && !cached_num_val && no_cache == 0)
 			  {
-			    PT_ERRORmf (this_parser, node, MSGCAT_SET_PARSER_SEMANTIC,
-					MSGCAT_SEMANTIC_SERIAL_ALTER_NO_OPTION, 0);
+			    if (comment != NULL)
+			      {
+			        node->info.serial.code = PT_SERIAL_COMMENT;
+			      }
 			  }
 
 		DBG_PRINT}}
+	| ALTER				/* 1 */
+	  SERIAL			/* 2 */
+	  serial_name			/* 3 */
+	  OWNER TO identifier		/* 4, 5, 6 */
+	  opt_comment_spec		/* 7 */
```

Comment:
What about removing opt_comment_spec?
Serial comments can be altered with the above form of ALTER SERIAL which changes serial options.

Reply:
Thank you for the code review.

If opt_comment_spec is removed from that syntax, OWNER TO and COMMENT cannot be used together as shown below
Therefore, OWNER TO .. opt_comment_spec is required.

ex)
ALTER SERIAL [serial_name] OWNER TO [user_name] COMMENT 'comment txt';

Reply:
Yes. I suggest not to use COMMENT block in the new ALTER SERIAL statement. 
It would be OK because comments can be altered in the old ALTER SERIAL statement.

Reply:
The new SERIAL syntax has been removed, and OWNER has been added to the old SERIAL syntax.
please confirm